### PR TITLE
libp11: 0.2.7 -> 0.4.4

### PR DESCRIPTION
**Pull request by ndowens**
###### Motivation for this change
Update

###### Things done

- [x] Tested using sandboxing
  ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS,
    or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] Linux
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by ndowens** [comment]
I dont know why travis is failing. It doesn't show libp11 depends on engine_pcks11 on my side at all

**Comment by ndowens** [comment]
According to https://github.com/OpenSC/libp11 this provides enginepkcs11 after 0.4.0

File: pkgs/development/libraries/libp11/default.nix [hunk 1]
```diff
@@ -1,21 +1,25 @@
...
+  makeFlags = [ "DESTDIR=$(out)" "PREFIX=" ];
+
   nativeBuildInputs = [ autoreconfHook pkgconfig libtool ];
   buildInputs = [ openssl ];
 
-  meta = {
-    homepage = http://www.opensc-project.org/libp11/;
+  meta = with stdenv.lib; {
+    homepage = https://github.com/OpenSC/libp11;
     license = "LGPL";
```
Comment (7c6f434c) [inline pkgs/development/libraries/libp11/default.nix]: The license ended up not cleaned up?

Comment (ndowens) [inline pkgs/development/libraries/libp11/default.nix]: woops :)

**Pull request merged**